spec/lib/generators/module_component/module_components_generator_spec.rb

```diff
@@ -0,0 +1,120 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require 'generators/module_component/module_component_generator'
+require 'generators/module/module_generator'
+
+RSpec.describe 'ModuleComponent', type: :generator do
```

Comment:
For all the tests here, should we also check that an expected value is inside the file (e.g. ensure ERB is parsing correctly)?

Reply:
That's probably a good idea

Reply:
We're not doing any line insertions in this generator, that's what module generator does when we're standing up the file structure. This just creates files from the templates.